README.md

```diff
@@ -133,6 +134,16 @@ Variables controlling cluster parameters (for amq_broker_instance_type = default
 | `amq_broker_cluster_replicated` | false | Set cluster to Replication mode |
 | `amq_broker_cluster_sharedstore` | false | Set cluster to Shared store mode |
 
+Supported Inventory: Host Variables
+------------
+
+Variables which are defined in inventory file specific for given host
+
+| Name              | Default Value       | Description          |
+|-------------------|---------------------|----------------------|
+| `external_address` | Not used by default | Useful if broker is running on internal network, but is VM is accessed from outside of this network |
+
```

Comment:
The variable needs to be named like amq_broker_external_address in order to prevent confusion with similar variables used in other roles.

Reply:
This is not related to AMQ broker at all, but to node, so adding amq_broker prefix does not make sense to me.
Unless you want to stress, that it is applicable for this role only.

Reply:
Resolved. amq_broker_ prefix added.